core/tee/tee_rpmb_fs.c

```diff
@@ -1803,7 +1803,7 @@ static TEE_Result fat_entry_dir_get_next(struct rpmb_fat_entry **fat_entry,
 #if (TRACE_LEVEL >= TRACE_FLOW)
 static void dump_fat(void)
 {
-	TEE_RESULT res;
+	TEE_Result res;
```

Comment:
by the way, could this also init `res` to something, as per optee_os convention?
I.e. `TEE_Result res = TEE_ERROR_SECURITY;`

Reply:
Uhm, that would make sense.

@jforissier should I make this change in this PR?

Reply:
Yes fine with me.